mesonbuild/options.py

```diff
@@ -1122,6 +1116,19 @@ def set_from_configure_command(self, D_args: T.List[str], U_args: T.List[str]) -
             if key in self.augments:
                 del self.augments[key]
                 dirty = True
+            else:
+                # TODO: For project options, "dropping an augment" means going
```

Comment:
```suggestion
                # TODO: For project options, "dropping an argument" means going
```

Reply:
No, this is how Jussi called options that are set on a subproject.